models/i-casework.js

```diff
@@ -33,11 +32,20 @@ module.exports = class CaseworkModel extends Model {
     return crypto.createHash('md5').update(date + config.icasework.secret).digest('hex');
   }
 
-  save() {
-    const options = this.requestConfig({});
-    options.form = this.prepare();
-    options.method = 'POST';
-
-    return this.request(options);
+  async save() {
+    try {
+      const data = await this.prepare();
+      const params = {
+        url: this.url(),
+        data,
+        timeout: config.icasework.timeout,
+        method: 'POST'
+      };
+      const response = await this._request(params);
+      return response;
+    } catch (error) {
+      console.error('Error saving data:', error);
```

Comment:
out of scope, it would be good to use the hof logger and ensure we have timestamps

Reply:
added hof logger now